packages/core/src/zip/__tests__/elementZipper.js

```diff
@@ -28,57 +29,61 @@ describe('Zipper', () => {
   it('zipper should correctly navigate up and down', () => {
     const zipper = elementZipper(fromJS(singleChild), 'children')
 
-    expect(zipper.value().get('kind')).toEqual('parent')
-    expect(data.isOfKind(childCollectionKind, zipper.down().value())).toBe(true)
+    expect(zip.node(zipper).get('kind')).toEqual('parent')
+    expect(data.isOfKind(childCollectionKind, zip.node(zip.down(zipper)))).toBe(
+      true
+    )
     expect(
-      zipper
-        .down()
-        .down()
-        .value()
-        .get('kind')
+      R.pipe(
```

Comment:
Not that critical, but @AleksandarDembovski please use `data.flow` instead of `R.pipe` in future.